pkgs/by-name/ho/hobbits/package.nix

```
@@ -46,11 +48,21 @@ stdenv.mkDerivation (finalAttrs: {
 
   env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.hostPlatform.isAarch64 "-Wno-error=narrowing";
 
+  postInstall = lib.optionalString stdenv.hostPlatform.isDarwin ''
+    mkdir -p $out/{Applications,bin}
+    mv $out/hobbits.app $out/Applications
+    wrapProgram $out/Applications/hobbits.app/Contents/MacOS/hobbits \
+      --prefix DYLD_LIBRARY_PATH : $out/Applications/hobbits.app/Contents/Frameworks
+    ln -s $out/Applications/hobbits.app/Contents/MacOS/hobbits $out/bin/hobbits
+    # Prevent wrapping
+    find $out/Applications -type f -name "*.dylib" -exec chmod -x {} \;
+  '';
+
   meta = {
     description = "Multi-platform GUI for bit-based analysis, processing, and visualization";
     homepage = "https://github.com/Mahlet-Inc/hobbits";
     license = lib.licenses.mit;
     maintainers = with lib.maintainers; [ sikmir ];
-    platforms = lib.platforms.linux;
+    platforms = lib.platforms.unix;
```

```suggestion
    platforms = with lib.platforms; linux ++ darwin;
```

Actually, Unix includes systems such as BSD, Darwin, Linux, SunOS, Cygwin, and Redox.